api/api-doc/storage_service.json

```diff
@@ -433,6 +433,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"path"
```

Comment:
There is too much going on in this commit, this should be split into like 4 different commits:
* add get_flat_token_ranges()
* make get_ranges() virtual
* change storage_service::effective_ownership()
* update API endpoint (maybe has to be folded into previous one if splitting is difficult)

Reply:
Done - I split it into three patches:
- locator/tablets: add tablet_map::get_flat_token_ranges()
- locator/effective_replication_map: make 'get_ranges(inet_address ep)' virtual
- storage_service/ownership/{keyspace}: handle requests when tablets are enabled